docs/integrations/destinations/s3.md

```diff
@@ -87,7 +87,7 @@ At this time, object-level permissions alone are not sufficient to successfully
 <!-- env:cloud -->
 
 :::note
-This authentication method is currently in the testing phase. To enable it for your workspace, please contact our Support Team.
+Currently, this feature is available only for the users in a Sales Assist workflow. Please contact your Solutions Engineer if you are interested in using this.
```

Comment:
Could we actually point them to the Talk to Sales form? They are not necessarily going to be actively talking with Sales already.
https://airbyte.com/company/talk-to-sales

Reply:
@nataliekwong let me know if the updated on is better.